src/components/Button/Button.tsx

```diff
@@ -265,7 +262,7 @@ const Button: VibeComponent<ButtonProps, unknown> & {
         className: classNames,
         name,
         onMouseUp,
-        style,
+        style: { minWidth: `${buttonWidth.current}px`, ...style },
```

Comment:
I intentionally put the extra style after in case someone overrode it or intending to override it, WDYT?